packages/concerto-core/lib/decoratormanager.js

```diff
@@ -252,6 +252,186 @@ class DecoratorManager {
+            decoratorCommandSet:dcms,
+            vocabularies:vocab
+        };
+    }
+
+    /**
+     * Parses the dict data into an array of decorator jsons
+     * @param {Object} decoratorDict the input dict
+     * @returns {Array<Object>} the parsed decorator command set array
+     */
+    static parseDecorators(decoratorDict){
+        let data = [];
+        Object.keys(decoratorDict).forEach((namespace)=>{
+            let nameOfDcs=namespace.split('@')[0];
+            let versionOfDcs=namespace.includes('@')?namespace.split('@')[1]:'1.0.0';
+            let dcsObjects=[];
+            let jsonData=decoratorDict[namespace];
+            jsonData.forEach((obj)=>{
+                let decos=JSON.parse(obj.dcs);
+                let target={
+                    '$class': 'org.accordproject.decoratorcommands.CommandTarget',
+                    'namespace':namespace
+                };
+                if (obj.declaration && obj.declaration!==''){
+                    target.declaration=obj.declaration;
+                }
+                if (obj.property && obj.property!==''){
+                    target.property=obj.property;
+                }
+                decos.forEach((dcs)=>{
+                    if (dcs.name!=='Term' && dcs.name!=='Term_description'){
+                        let decotatorObj={
+                            '$class': 'concerto.metamodel@1.0.0.Decorator',
+                            'name': dcs.name,
+                        };
+                        if (dcs.arguments){
+                            let args=dcs.arguments.map((arg)=>{
+                                return {
+                                    '$class':arg.$class,
+                                    'value':arg.value
+                                };
+                            });
+                            decotatorObj.arguments=args;
+                        }
+                        let dcsObject = {
+                            '$class': 'org.accordproject.decoratorcommands.Command',
+                            'type': 'UPSERT',
+                            'target': target,
+                            'decorator': decotatorObj,
+                        };
+                        dcsObjects.push(dcsObject);
+                    }
+                });
+            });
+            let dcmsForNamespace={
+                '$class': 'org.accordproject.decoratorcommands.DecoratorCommandSet',
+                'name': nameOfDcs,
+                'version': versionOfDcs,
+                'commands': dcsObjects
+            };
+            data.push(dcmsForNamespace);
+        });
+        return data;
+    }
+    /**
+     * Parses the dict data into an array of decorator jsons
+     * @param {Object} decoratorDict the input dict
+     * @param {String} locale locale for target vocab
+     * @returns {Array<Object>} the parsed decorator command set array
+     */
+    static parseVocabs(decoratorDict,locale){
```

Comment:
```suggestion
    static parseVocabularies(decoratorDict,locale){
```

The pattern in this repo is to avoid abbreviations in function names